sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -225,7 +225,8 @@ public async Task CopyDocumentModelTo(bool useTokenCredential)
 
             CollectionAssert.AreEquivalent(tags, model.Tags);
 
-            AssertDocumentTypeDictionariesAreEquivalent(sourceModel.DocumentTypes, model.DocumentTypes);
+            // (TODO) Reenable validation once the following service issue has been fixed: https://github.com/Azure/azure-sdk-for-net/issues/37172
```

Comment:
nit: By default, Visual Studio's Task List only recognizes "TODO" if it's at the start of the comment, i.e., without the opening parenthesis.

Reply:
Fixed. To be frank, I wasn't even aware that VS had that feature.

Reply:
Something weird is going on. I updated my branch in GitHub with a new commit but it's not being included in this PR.

Reply:
Since it's not a major issue, I'll just merge it and fix it in another PR.